android/build.gradle

```diff
@@ -112,6 +112,8 @@ android {
 
   packagingOptions {
     excludes = ["**/libc++_shared.so"]
+    exclude "META-INF/**"
+    pickFirst '**/*.so'
```

Comment:
why `**/*.so`? why not just the `META-INF/**`?

Reply:
I've added this mostly because if there are some other .so files they will conflict as well (in this case when you're integrating OpenCV for another frame processor plugin.

Reply:
> thanks for your PR! Left a comment there because I don't see how that's related.. also, could you tell me how you can reproduce the build error? I've never seen it before and I am not aware of any side-effects of ignoring META-INF...

It happened to me a couple of times when using a combination of Hermes + Detox + Vision Camera Library, it might not be directly related to this plugin itself, but it ended up working nicely 🤔